static/app/views/performance/newTraceDetails/traceDrawer/details/styles.tsx

```diff
@@ -536,15 +560,32 @@ function SectionCard({
   const contentItems = items.map(item => ({item, ...itemProps}));
 
   return (
-    <KeyValueData.Card
-      title={title}
-      contentItems={contentItems}
-      sortAlphabetically={sortAlphabetically}
-      truncateLength={disableTruncate ? Infinity : 5}
-    />
+    <CardWrapper>
+      <KeyValueData.Card
+        title={title}
+        contentItems={contentItems}
+        sortAlphabetically={sortAlphabetically}
+        truncateLength={disableTruncate ? Infinity : 5}
+      />
+    </CardWrapper>
   );
 }
 
+// This is trace-view specific styling. The card is rendered in a number of different places
+// with tests failing otherwise, since @container queries are not supported by the version of
+// jsdom currently used by jest.
+const CardWrapper = styled('div')`
+  ${CardPanel} {
+    container-type: inline-size;
+  }
+
+  ${Subject} {
+    @container (width < 350px) {
+      max-width: 200px;
+    }
+  }
+`;
```

Comment:
Should this be part of this PR?

Reply:
Oh no, I need to remove this, I ammended a commit and messed it up

Reply:
No, I removed it. 4 day weekends hit hard